Couple-Memory_backend/config/db.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config(); // Load environment variables from .env file
+const mongoose = require('mongoose');
+const mongoURI = `mongodb+srv://saura8668:${process.env.USER_KEY}@cluster0.c48azqk.mongodb.net/viewcount`;
```

Comment:
I think here is missing the .env File

Isnt it possible to run mongo db on this same server?
If not, I rather configure this.